backend/fms_core/models/run_type.py

```diff
@@ -50,13 +53,12 @@ def save(self, *args, **kwargs):
         self.full_clean()
         super().save(*args, **kwargs)  # Save the object
 
-
-    # This method should be moved out of ExperimentType when Workflows are implemented in FreezeMan
+    # REPLACE and move to procotol
     @property
     def get_protocols_dict(self):
```

Comment:
Same comment as the 2 previous ones.. shouldnt we fetch this from the db from now on ?